src/common/ServerAPI.jsx

```diff
@@ -32,53 +32,42 @@ async function loginAccount(email, password) {
   const masterKey = await deriveMasterKey(email, password);
   const passwordHash = await deriveMasterPasswordHash(password, masterKey);
 
-  try {
-      const response = await axios.post(
-          LOGIN_ACCOUNT_URL,
-          {
-              email: email,
-              password: passwordHash,
-          },
-      );
-
-      const encryptedSymmetricKey = response.data.encrypted_symmetric_key;
-      const symmetricKey = await decryptSymmetricKey(encryptedSymmetricKey, masterKey);
-
-      return Promise.resolve(symmetricKey);
+  const response = await axios.post(
+    LOGIN_ACCOUNT_URL,
+    {
+        email: email,
+        password: passwordHash,
```

Comment:
Nit: It looks like you updated to using 2 spaces instead of 4. You don't have to update these two lines, but it looks like it is still 4 spaces here.

Reply:
I think my local linting isn't in sync with the project - I'll change this before I merge